yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -19,14 +19,14 @@
       <div class= "border">
       {% if edit %}
         <div class="edit_button">
+        <form method="get" action="{% url 'edit_page' workflow.id version.id %}">
+            {% csrf_token %}
+            <button type="submit"> Edit Workflow</button>
+        </form>
+        <div>
+            <button onclick="deleteWorkflow()">Delete Workflow</button>
+        </div>
       {% endif %}
-            <form method="get" action="{% url 'edit_page' workflow.id version.id %}">
```

Comment:
you might want to move this endif to remove all the javascript. look at what i did in this pr https://github.com/aniehuser/senior-design-group10/pull/162/files

Reply:
@aniehuser I merged you're fix with this PR

Reply:
ok sweet